src/server/scripts/Northrend/FrozenHalls/HallsOfReflection/halls_of_reflection.cpp

```diff
@@ -272,7 +330,7 @@ class npc_hor_leader : public CreatureScript
                 case EVENT_INTRO_A2_3:
                     pInstance->HandleGameObject(pInstance->GetGuidData(GO_FROSTMOURNE), true);
                     me->CastSpell(me, SPELL_FROSTMOURNE_SPAWN_SOUND, true);
-                    me->CastSpell(me, SPELL_ARCANE_CAST_VISUAL, false);
+                    //me->CastSpell(me, SPELL_ARCANE_CAST_VISUAL, false);// tirar assim que acabar de trabalhar na horda
```

Comment:
leftover

Reply:
> leftover

in that case she has to launch a skill, I'm just waiting for the blizzard servers to come back for me to sniff. I left it commented because she really has to cast a skill.